AllReadyApp/Web-App/AllReady/Views/Shared/_LayoutMainPage.cshtml

```diff
@@ -65,10 +66,17 @@
         <script src="//ajax.aspnetcdn.com/ajax/bootstrap/3.0.0/bootstrap.min.js"
                 asp-fallback-src="~/lib/bootstrap/dist/js/bootstrap.min.js"
                 asp-fallback-test="window.jQuery && window.jQuery.fn && window.jQuery.fn.modal">
+        
+        </script>
+        <script src="//cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.6/moment.min.js"
+                asp-fallback-src="~/lib/moment/moment.js"
+                asp-fallback-test="window.moment">
         </script>
     </environment>
+
     <script src="~/js/site.js"></script>
-    <script src="~/js/index.js"></script>
+    <script src="~/js/ko.filterableList.js" asp-append-version="true"></script>

```

Comment:
You won't need this line because you don't actually need any of the client side filtering functionality.